bgpd/bgp_route.c

```diff
@@ -8185,7 +8184,7 @@ void route_vty_out_tag(struct vty *vty, const struct prefix *p,
 	/* print prefix and mask */
 	if (json == NULL) {
 		if (!display)
-			route_vty_out_route(p, vty, NULL);
+			route_vty_out_route(p, vty, NULL, 0 /*wide*/);
```

Comment:
true/false please for booleans instead of 0/1.

Reply:
addressed in the latest commit